### ISSUE TITLE
Tach report fails on Windows paths

### ISSUE THREAD

**Issue by aawilson** [gauge-sh/tach#361]
See https://github.com/gauge-sh/tach/issues/326.

Running `tach report` on a path in Windows (at least in PowerShell, probably also in cmd.exe) raises the following error. The same path in WSL (bash) successfully generates a report.
```
Report failed: Filesystem error: No matching source root found for filepath: "\\\\?\\C:\\users\\aawilson\\projects\\someproject\\some\\path"
```

**Comment by emdoyle** [comment]
This should be fixed in `0.14.1`! Please let me know if the issue persists @aawilson

**Comment by aawilson** [comment]
Thanks for all the work, guys!
